inscriptions/ISic002750.xml

```diff
@@ -193,26 +197,26 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI332879">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#FP">
                 <ab>
-                    <lb n="1"/>
-
-
-                    ἐνθάδε
-                    <unclear>κ</unclear>
-                    <supplied reason="lost">εῖται</supplied>
-                    <lb n="2"/>Νώνιος Π
+                    <lb n="1"/><w>Ἐνθάδε</w>
+                    <w><unclear>κ</unclear>
+                    <supplied reason="lost">εῖται</supplied></w>
+                    <lb n="2"/> <persName type="attested"><name type="nomen">Νώνιος</name> <name type="cognomen">Π
```

Comment:
The name needs treating slightly differently:
`<name type="cognomen"><seg part="I">Π<unclear>ι</unclear></seg><gap reason="lost" extent="unknown" unit="character"/></name>`
In other words, you're right to put the gap inside the name, but we also use seg with part (I=Initial, M=Middle, F=Final) to describe the surviving segment of the name.